packages/node_modules/@node-red/nodes/core/network/21-httprequest.js

```diff
@@ -586,6 +586,10 @@ in your Node-RED user directory (${RED.settings.userDir}).
                     opts.https.certificate = opts.https.cert;
                     delete opts.https.cert;
                 }
+                if (opts.https.ALPNProtocols) {
```

Comment:
```suggestion
                // The got library uses a different case for some https properties compared to the
                // standard node tls options object.
                if (opts.https.ALPNProtocols) {
```

Suggest adding an inline comment for why this change is needed - so if we move off got in the future, we can spot these edge cases more easily.